generators/client/index.js

```diff
@@ -187,19 +224,28 @@ module.exports = JhipsterClientGenerator.extend({
             insight.track('generator', 'app');
             insight.track('app/useSass', this.useSass);
             insight.track('app/enableTranslation', this.enableTranslation);
+            insight.track('app/nativeLanguage', this.nativeLanguage);
+            insight.track('app/languages', this.languages);
         },
 
         configureGlobal: function () {
             // Application name modified, using each technology's conventions
             this.angularAppName = this.getAngularAppName();
-            this.camelizedBaseName = _.camelize(this.baseName);
-            this.slugifiedBaseName = _.slugify(this.baseName);
+            this.camelizedBaseName = _s.camelize(this.baseName);
+            this.slugifiedBaseName = _s.slugify(this.baseName);
             this.lowercaseBaseName = this.baseName.toLowerCase();
         },
 
         saveConfig: function () {
             this.config.set('useSass', this.useSass);
             this.config.set('enableTranslation', this.enableTranslation);
+            if (this.enableTranslation) {
+                this.config.set('nativeLanguage', this.nativeLanguage);
+                this.config.set('languages', this.languages);
+            } else {
```

Comment:
Yes it will be undefined by default but it will not reset the current languages and nativeLanguage options if one sets the enableTranslate to false. Maybe this is not needed though it was handy when I manually modified my yo-rc.json.